Apps/WebClient/src/ClientApp/src/components/modal/AppTourComponent.vue

```diff
@@ -0,0 +1,202 @@
+﻿<script lang="ts">
+import { BCarousel } from "bootstrap-vue";
+import Vue from "vue";
+import { Component, Ref } from "vue-property-decorator";
+import { Getter } from "vuex-class";
+
+import type { WebClientConfiguration } from "@/models/configData";
+
+interface TourSlide {
+    // Title to be displayed for the slide.
+    title: string;
+    // Description to be displayed for the slide.
+    description?: string;
+    // Image or Gif to be displayed for the slide.
+    imageUri: string;
+    // Optional alt text for the image.
+    imageAlt?: string;
+}
+
+@Component
+export default class AppTourComponent extends Vue {
+    @Getter("webClient", { namespace: "config" })
+    webClient!: WebClientConfiguration;
+
+    @Ref("tourCarousel")
+    readonly tourCarousel!: BCarousel;
+
+    @Getter("isMobile")
+    isMobile!: boolean;
+
+    private mobileSlides: TourSlide[] = [
+        // TODO: Final configuration in AB#15392
+        {
+            title: "MB-Add a Quick Link",
```

Comment:
This title is displayed to the user.

```suggestion
            title: "Add a Quick Link",
```